packages/core/src/base-axis-chart.ts

```
@@ -594,56 +591,7 @@ export class BaseAxisChart extends BaseChart {
 	/**************************************
 	 *  Events & User interactions        *
 	 *************************************/
-
 	addDataPointEventListener() {
-		const self = this;
-		const { accessibility } = this.options;
-
-		this.svg.selectAll("rect")
-			.on("click", function(d) {
-				self.dispatchEvent("bar-onClick", d);
-			})
-			.on("mouseover", function(d) {
-				select(this)
-					.attr("stroke-width", Configuration.bars.mouseover.strokeWidth)
-					.attr("stroke", self.colorScale[d.datasetLabel](d.label))
-					.attr("stroke-opacity", Configuration.bars.mouseover.strokeOpacity);
-
-				self.showTooltip(d, this);
-				self.reduceOpacity(this);
-			})
-			.on("mousemove", function(d) {
-				const tooltipRef = select(self.holder).select("div.chart-tooltip");
-
-				const relativeMousePosition = mouse(self.holder as HTMLElement);
-				tooltipRef.style("left", relativeMousePosition[0] + Configuration.tooltip.magicLeft2 + "px")
-					.style("top", relativeMousePosition[1] + "px");
-			})
-			.on("mouseout", function(d) {
-				const { strokeWidth, strokeWidthAccessible } = Configuration.bars.mouseout;
-				select(this)
-					.attr("stroke-width", accessibility ? strokeWidthAccessible : strokeWidth)
-					.attr("stroke", accessibility ? self.colorScale[d.datasetLabel](d.label) : "none")
-					.attr("stroke-opacity", Configuration.bars.mouseout.strokeOpacity);
-
-				self.hideTooltip();
-			});
-
-		this.svg.selectAll("circle.dot")
-			.on("mouseover", function(d) {
-				select(this)
-					.attr("stroke", self.colorScale[d.datasetLabel](d.label))
-					.attr("stroke-opacity", Configuration.lines.points.mouseover.strokeOpacity);
-
-				self.showTooltip(d, this);
-				self.reduceOpacity(this);
-			})
-			.on("mouseout", function(d) {
-				select(this)
-					.attr("stroke", self.colorScale[d.datasetLabel](d.label))
-					.attr("stroke-opacity", Configuration.lines.points.mouseout.strokeOpacity);
-
-				self.hideTooltip();
-			});
+		return;
```

This function should throw an error saying:
```js
console.warn("You should implement your own `addDataPointEventListener()` function.");
```

No return needed btw.